.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -11,7 +11,7 @@ assignees: ''
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
 Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
 
-This repository is **ONLY** for PowerShell Core 6 issues.
+This repository is **ONLY** for PowerShell Core 6 and PowerShell 7 issues.
```

Comment:
Why not just say "PowerShell Core" (version 6 or higher) issues", or, with a view to possibly dropping the "Core" qualifier soonish, "for issues relating to PowerShell version 6 and above"

Reply:
Officially, we're dropping `Core` from the name hence forth.  Current thinking is to stay on 7.x so I don't expect a PS8 for some time if ever

Reply:
Thanks for clarifying, @SteveL-MSFT - how does the "vNext" you were referring to [here](https://github.com/PowerShell/PowerShell/issues/1995#issuecomment-552533786) fit into the picture? Were just referring to 7.1?

Reply:
```suggestion
This repository is **ONLY** for PowerShell Core 6 and PowerShell 7+ issues.
```